libs/tokens/src/utils/tokenMapToListWithLogo.ts

```diff
@@ -7,6 +7,6 @@ import { TokensMap } from '../types'
  */
 export function tokenMapToListWithLogo(tokenMap: TokensMap): TokenWithLogo[] {
   return Object.values(tokenMap)
-    .sort((a, b) => (a.symbol > b.symbol ? 1 : -1))
+    .sort((a, b) => (a.symbol === b.symbol ? 0 : a.symbol > b.symbol ? 1 : -1))
```

Comment:
Fix for sorting.
We were not accounting for when symbols are identical.

Reply:
Ups we missed this!

Reply:
Actually, [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) is what we should use for strings.

Reply:
Wow! I didn't even know about it